app/slashcommands/command_share.go

```diff
@@ -52,14 +52,11 @@ func (sp *ShareProvider) GetCommand(a *app.App, T goi18n.TranslateFunc) *model.C
 
 	status := model.NewAutocompleteData("status", "", "Displays status for this shared channel")
 
-	sync := model.NewAutocompleteData("sync", "", "Forces a sync with remote instances this shared channel")
```

Comment:
Not sure if this is intentional, or a bad merge?

Reply:
My understanding is that the manual sync command now is not needed, since we are now automatically handling that. I can put it back, will defer to @wiggin77 on this

Reply:
Discussed this during planning and the decision was to put the manual one back for an escape hatch when things fail to work.